cmd/bosun/sched/check.go

```diff
@@ -185,9 +186,63 @@ func (s *Schedule) RunHistory(r *RunHistory) {
 	if checkNotify && s.nc != nil {
 		s.nc <- true
 	}
+	s.CollectStates()
 	s.Save()
 }
 
+// CollectStates sends various state information to bosun with collect.
+func (s *Schedule) CollectStates() {
+	metadata.AddMetricMeta(
+		"bosun.alerts.current_severity", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by current severity.")
+	metadata.AddMetricMeta(
+		"bosun.alerts.last_abnormal_severity", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by last abnormal severity.")
+	metadata.AddMetricMeta(
+		"bosun.alerts.acknowledgement_status", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by acknowledgement status.")
+	metadata.AddMetricMeta(
+		"bosun.alerts.active_status", metadata.Gauge, metadata.Alert,
+		"The number of open alerts by active status.")
+	for _, state := range s.status {
+		if !state.Open {
+			return
+		}
+		// The tagset of the alert is not included because there is no way to
+		// store the string of a group in OpenTSBD in a parsable way. This is
+		// because any delimiter we chose could also be part of a tag key or tag
+		// value.
+		ts := opentsdb.TagSet{"alert": state.Alert}
+		severity := state.Status().String()
+		lastAbnormal := state.AbnormalStatus().String()
+		var err error
+		err = collect.Put("alerts.current_severity", ts.Copy().Merge(opentsdb.TagSet{"severity": severity}), 1.0)
+		if err != nil {
+			log.Println(err)
+		}
+		err = collect.Put("alerts.last_abnormal_severity", ts.Copy().Merge(opentsdb.TagSet{"severity": lastAbnormal}), 1.0)
+		if err != nil {
+			log.Println(err)
+		}
+		ackString := "acknowledged"
+		if state.NeedAck {
+			ackString = "unacknowledged"
+		}
+		err = collect.Put("alerts.acknowledgement_status", ts.Copy().Merge(opentsdb.TagSet{"status": ackString}), 1.0)
```

Comment:
"Again, why not use a value of 0 or 1 to indicate ack'd, instead of tags? Same for active below." I want to count them, and if I use zero I can't count via sum aggregation things with a zero value.